modules/checkout/src/Annotation/CommerceCheckoutPane.php

```diff
@@ -1,4 +1,5 @@
 <?php
+// @codingStandardsIgnoreFile
```

Comment:
I agree with your changes. I will fix up as soon as possibble.

man. 9. maj 2016 kl. 20.46 skrev Harings Rob <notifications@github.com>:

> In modules/checkout/src/Annotation/CommerceCheckoutPane.php
> <https://github.com/drupalcommerce/commerce/pull/376#discussion_r62551358>
> :
>
> > @@ -1,4 +1,5 @@
> >  <?php
> > +// @codingStandardsIgnoreFile
>
> I thinks we can quickly refactor this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/drupalcommerce/commerce/pull/376/files/759e1db0430dc4bc7435b26468a4a6a45a636a34..8413c5606ce12fe568a5f3caf4c54916d50fe82b#r62551358>
>